### DISCUSSION TITLE
All videos showed No Results Found after clicking Scrape All button

### DISCUSSION THREAD

**Discussion by howietung** [stashapp/stash#5085] [Q&A]
Hello everyone
I am sorry that I need to ask for some help, I followed the manuel and installed the scraper but when I clicked Scrape All button under the scenes, all the videos showed No Results Found, I am not sure what I did wrong. Does it suppose to help you to match all the video files to Metadata after clicking the Scrape All button? I checked the log but no error messages for it. It would be really nice if someone can help me out!! Thank you!!

**Comment by everforyou** [comment]
Hello there,
It sounds frustrating that you're getting "No Results Found" when you click the "Scrape All" button. Here are a few things you can check to troubleshoot the issue:

- File Naming: Make sure the video files are named correctly and match the expected format for the metadata.
- Metadata Configuration: Double-check the settings for the metadata source. Ensure that it's properly configured and accessible.
- Permissions: Verify that the scraper has the necessary permissions to access both the video files and the metadata source.
- Review Logs: Even if there are no error messages, look for any warnings or informational messages in the logs that might indicate what’s going wrong.
- Update the Scraper: Ensure that your scraper is up to date, as updates can fix bugs and improve functionality.

Thank you!!

**Reply by howietung** [reply]
Thank you for helping me out 
Can you help me and tell me what format are the expected format?
Also how to verify that the scraper has the necessary permissions?
Thank you so so much!!

**Reply by everforyou** [reply]
For the expected format, video files are usually in formats like MP4, MKV, or AVI. Make sure the file extensions match what the scraper recognizes.

To verify the scraper's permissions, check that the user account running the scraper has read access to the video files and directories. Also, if your metadata source is a database or API, ensure the credentials used have the right permissions to access it.

**Reply by HereIam80** [reply]
I honestly have no idea what you are talking about. The file extension doesn't matter at all for scraping and even the file name or folder structure doesn't matter if you are using a stashBox endpoint as the source with the "Scrape all" button.

**Comment by HereIam80** [comment]
What scraper are you using?
Did you generate phashes?

**Reply by howietung** [reply]
I am not sure if they only use 1 scraper or multiple scrapers at once when you click Scrape All button, because I installed many scrapers. And I just realized I didn't generate phashes, so I am doing it now. I will try again after phashes are generated. 
Thank you for helping me out!!

**Reply by HereIam80** [reply]
I advise you to go through this guide: https://docs.stashapp.cc/beginner-guides/guide-to-scraping/ to help you out.
Scrape all doesn't mean it uses multiple scrapers. If you are in the tagger view you have to select a source in the upper left. That is what matters. If you have selected "Auto-Tag" for example that basically does nothing. You have to specify the right scraper for the right content or use a stash-box endpoint.
